ocs_ci/ocs/resources/mcg.py

```diff
@@ -947,58 +954,55 @@ def retrieve_noobaa_cli_binary(self):
             AssertionError: In the case CLI binary doesn't exist.
 
         """
-        if not os.path.isfile(constants.NOOBAA_OPERATOR_LOCAL_CLI_PATH):
-            logger.info(
-                f"The MCG CLI binary could not be found in {constants.NOOBAA_OPERATOR_LOCAL_CLI_PATH},"
-                " attempting to copy it from the quay.io"
-            )
-            semantic_version = version.get_semantic_ocs_version_from_config()
-            remote_path = self.get_architecture_path()
-            remote_mcg_cli_basename = os.path.basename(remote_path)
-            local_mcg_cli_dir = os.path.dirname(
-                constants.NOOBAA_OPERATOR_LOCAL_CLI_PATH
-            )
-            if (
-                config.DEPLOYMENT["live_deployment"]
-                and semantic_version >= version.VERSION_4_13
-            ):
-                image = f"registry.redhat.io/odf4/mcg-cli-rhel9:v{semantic_version}"
-            else:
-                image = f"quay.io/rhceph-dev/mcg-cli:{get_ocs_build_number()}"
+        logger.info(
+            f"The MCG CLI binary could not be found in {constants.NOOBAA_OPERATOR_LOCAL_CLI_PATH},"
+            " attempting to copy it from the quay.io"
+        )
+        semantic_version = version.get_semantic_ocs_version_from_config()
+        remote_path = self.get_architecture_path()
+        remote_mcg_cli_basename = os.path.basename(remote_path)
+        local_mcg_cli_dir = os.path.dirname(constants.NOOBAA_OPERATOR_LOCAL_CLI_PATH)
+        if (
+            config.DEPLOYMENT["live_deployment"]
+            and semantic_version >= version.VERSION_4_13
+        ):
+            image = f"registry.redhat.io/odf4/mcg-cli-rhel9:v{semantic_version}"
```

Comment:
can we save the URL prefix in a constant file? that way it should be easier to change it later if needed

Reply:
discussed offline and it will be addressed in a follow up PR